### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by sumlin**

### CODE REVIEW

**Comment by TVolden** [comment]
Looks like a build failure. Can you build it with maven locally?

**Comment by sumlin** [comment]
Sometimes. If I have IDEA's classes, it can be built from maven too.

Anyway, I've fixed it.

**Comment by sumlin** [comment]
@TVolden  All tests are passed locally, the PR can be merged.

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/ChargingProfile.java [hunk 1]
```diff
@@ -49,38 +48,47 @@
...
+                           ChargingProfileKindType chargingProfileKind, ChargingSchedule chargingSchedule) {
         this.chargingProfileId = chargingProfileId;
         this.stackLevel = stackLevel;
         this.chargingProfilePurpose = chargingProfilePurpose;
         this.chargingProfileKind = chargingProfileKind;
         this.chargingSchedule = chargingSchedule;
     }
 
-    public ChargingProfile(Integer chargingProfileId, Integer stackLevel, ChargingProfilePurposeType chargingProfilePurpose, ChargingProfileKindType chargingProfileKind) {
+    public ChargingProfile(int chargingProfileId, Integer transactionId, int stackLevel,
```
Comment (TVolden) [inline ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/ChargingProfile.java]:
```suggestion
    /**
     * Alternative constructor, with required and optional fields.
     *
     * @param chargingProfileId required
     * @param transactionId optional
     * @param stackLevel required
     * @param chargingProfilePurpose required
     * @param chargingProfileKind required
     * @param recurrencyKind optional
     * @param validFrom optional
     * @param validTo optional
     * @param chargingSchedule required
     */
    public ChargingProfile(int chargingProfileId, Integer transactionId, int stackLevel,
```

**Comment by TVolden** [comment]
Boink boink, any news @sumlin?

**Comment by TVolden** [comment]
I'm going to close this, since it's inactive and way behind by now. @sumlin please don't hesitate to open a new PR when you're ready. :)

**Pull request closed without merging**